frontend/src/Settings/General/UpdateSettings.js

```diff
@@ -83,8 +83,10 @@ function UpdateSettings(props) {
                 type={inputTypes.CHECK}
                 name="updateAutomatically"
                 helpText={translate('UpdateAutomaticallyHelpText')}
+                helpTextWarning={updateMechanism.value === 'docker' ? translate('AutomaticUpdatesAreDisabledWhenUsingTheDockerUpdateMechanism') : undefined}
```

Comment:
This translation key is too verbose, something like `AutomaticUpdatedDisabledDocker` would be more appropriate, I've already discussed downstream syncing up with Q.

Reply:
We don't want to use the English Translation as a key, so we shouldn't convert the English translation to a key.